packages/validation/src/Questionary/numberInput.ts

```diff
@@ -14,12 +14,26 @@ export const numberInputQuestionValidationSchema = (
     valueScheme = valueScheme.required('This is a required field');
   }
 
-  if (config.numberValueConstraint === NumberValueConstraint.ONLY_NEGATIVE) {
-    valueScheme = valueScheme.negative('Value must be a negative number');
-  }
-
-  if (config.numberValueConstraint === NumberValueConstraint.ONLY_POSITIVE) {
-    valueScheme = valueScheme.positive('Value must be a positive number');
+  switch (config.numberValueConstraint) {
+    case NumberValueConstraint.ONLY_NEGATIVE:
+      valueScheme = valueScheme.negative('Value must be a negative number');
+      break;
+
+    case NumberValueConstraint.ONLY_POSITIVE:
+      valueScheme = valueScheme.positive('Value must be a positive number');
+      break;
+
+    case NumberValueConstraint.ONLY_NEGATIVE_INTEGER:
```

Comment:
It looks to me like `NumberValueConstraint` is an enum [defined in the backend](https://github.com/UserOfficeProject/user-office-backend/blob/cf128f6003b6e683b33ca80550291a3007984c21/src/resolvers/types/FieldConfig.ts#L154), but currently doesn't have the _INTEGER options. Do we need a separate linked PR there to add these new values?

Reply:
I will do, but I'm not opening them until I can reference the new `validation` package, which requires this to be released first.

Reply:
So how is validation going to compile then if it doesn't have a definition of these _INTEGER symbols?

Reply:
The enum for `NumberValueConstraint` is passed in as an argument (`any` type) so this resolves to `undefined`.